app/templates/main/breadcrumb.html

```diff
@@ -2,7 +2,7 @@
 {% set transferring_body_url = browse_url ~ "?transferring_body_id=" %}
 {% set series_url = browse_url ~ "?series_id=" %}
 {% set consignment_url = browse_url ~ "?consignment_id=" %}
-<div class="govuk-breadcrumbs">
+<div class="govuk-breadcrumbs  {{ 'govuk-breadcrumbs--file' if record_breadcrumbs }}">
```

Comment:
instead of introducing `record_breadcrumbs` we could just do if `if "File" in dict`

corresponding to `File` set in the top of record.html
```
{% set breadcrumbs = {
    "everything":"Everything",
    "body": [breadcrumb_values[0]["transferring_body_id"], breadcrumb_values[1]["transferring_body"]],
    "series": [breadcrumb_values[2]["series_id"], breadcrumb_values[3]["series"]],
    "consignment": [breadcrumb_values[4]["consignment_id"], breadcrumb_values[5]["consignment_reference"]],
    "File": breadcrumb_values[6]['file_name']
} %}
```

which will keep things consistent with how we check for values inside the breadcrumb dict below.

Also, whilst might be nice to rename "File" to "record" for consistency

Reply:
updated.  Not sure on last part of comment, so renamed to 'record', if it's incorrect, will change it @anthonyhashemi